resources/warnings.json

```diff
@@ -7338,8 +7338,8 @@
     "url": "https://jenkins.io/security/advisory/2020-03-09/#SECURITY-1523",
     "versions": [
       {
-        "lastVersion": "1.3.1",
-        "pattern": ".*"
+        "lastVersion": "312.vcde1075036ef",
+        "pattern": "(1[.][1-9]|1[.]1[0-9]|1[.]2[0-9]|1[.]3[0-3]|314)(|[.-].+)"
```

Comment:
All `1.*` can be collapsed; 309 was improperly removed in the update; and we match affected, not fixed, so 312 instead of 314.

```suggestion
        "lastVersion": "312.vcde1075036ef",
        "pattern": "(1|309|312)(|[.-].+)"
```